systest/tests/checkpoint_test.go

```diff
@@ -169,15 +169,15 @@ func TestCheckpoint(t *testing.T) {
 	}
 
 	tctx.Log.Infow("waiting for all miners to be smeshing", "last epoch", checkpointEpoch+2)
-	ensureSmeshing(t, tctx, cl, checkpointEpoch+2)
+	ensureSmeshing(t, tctx, cl, checkpointEpoch+4)
 
 	// increase the cluster size to the original test size
 	tctx.Log.Info("cluster size changed to ", size)
 	tctx.ClusterSize = size
 	require.NoError(t, cl.AddSmeshers(tctx, addedLater))
 
 	tctx.Log.Infow("waiting for all miners to be smeshing", "last epoch", lastEpoch)
-	ensureSmeshing(t, tctx, cl, lastEpoch)
+	ensureSmeshing(t, tctx, cl, lastEpoch+6)
```

Comment:
Same here: I think we can change the value of `lastEpoch` directly to `14` instead of `8`.